### PR TITLE
Docs: Update validator hardware recommendations

### PR DESCRIPTION
**Pull request by t-nelson**
#### Problem

Validator hardware recommendations are out of date

#### Summary of Changes

Update and generalize them

### CODE REVIEW

File: docs/src/running-validator/validator-reqs.md [hunk 1]
```diff
@@ -2,35 +2,31 @@
...
+  - 128GB, or more
+  - Motherboard with 256GB capacity suggested
+- Disk
+  - PCIe gen3 x4 NVME SSD, or better
+  - Accounts: 500GB, or larger. High TBW
+  - Ledger: 1TB or larger. High TBW suggested
+  - OS: (Optional) 500GB, or larger. SATA OK
+  - The OS may be installed on the ledger disk, though testing has shown better
+performance with the ledger on its own disk
+  - Accounts and ledger _can_ be stored on the same disk however, due to high
```
Comment (CriesofCarrots) [inline docs/src/running-validator/validator-reqs.md:23]:
```suggestion
  - Accounts and ledger _can_ be stored on the same disk, however due to high
```

File: docs/src/running-validator/validator-reqs.md [hunk 2]
```diff
@@ -42,7 +38,7 @@ your own internal usage. This use case includes exchanges and services built on
 Solana.
 
 In fact, the offical mainnet-beta API nodes are currently (Oct. 2020) run on GCE
-`n1-standard-32` (32 vCPUs, 120 GB memory) instances with 2048 GB SSD for
+`n2-standard-32` (32 vCPUs, 128 GB memory) instances with 2048 GB SSD for
```
Comment (ryoqun) [inline docs/src/running-validator/validator-reqs.md:41]: good eyes. :)

File: docs/src/running-validator/validator-reqs.md [hunk 3]
```diff
@@ -2,35 +2,31 @@
...
+  - 2.8GHz, or faster
+  - AVX2 instruction support
+  - Support for AVX512f and/or SHA-NI instructions is helpful
+  - The AMD Threadripper Zen3 series is popular with the validator community
+- RAM
+  - 128GB, or more
+  - Motherboard with 256GB capacity suggested
+- Disk
+  - PCIe gen3 x4 NVME SSD, or better
+  - Accounts: 500GB, or larger. High TBW
```
Comment (ryoqun) [inline docs/src/running-validator/validator-reqs.md:18]: TBW = total band width? is it common?

File: docs/src/running-validator/validator-reqs.md [hunk 4]
```diff
@@ -2,35 +2,31 @@
...
-
-- CPU Recommendations
-  - We recommend a CPU with the highest number of cores as possible. AMD Threadripper or Intel Server \(Xeon\) CPUs are fine.
-  - We recommend AMD Threadripper as you get a larger number of cores for parallelization compared to Intel.
-  - Threadripper also has a cost-per-core advantage and a greater number of PCIe lanes compared to the equivalent Intel part. PoH \(Proof of History\) is based on sha256 and Threadripper also supports sha256 hardware instructions.
-- SSD size and I/O style \(SATA vs NVMe/M.2\) for a validator
-  - Minimum example - Samsung 860 Evo 2TB
-  - Mid-range example - Samsung 860 Evo 4TB
-  - High-end example - Samsung 860 Evo 4TB
+## Hardware Recommendations
```
Comment (ryoqun) [inline docs/src/running-validator/validator-reqs.md:5]: These specific specs looks decent. But @sakridge might have more thoughts than me. :)

File: docs/src/running-validator/validator-reqs.md (hunk 3 above)
Comment (sakridge) [reply docs/src/running-validator/validator-reqs.md:18]: total bytes written, it's a common acronym related to SSD drive life.

File: docs/src/running-validator/validator-reqs.md [hunk 5]
```diff
@@ -2,35 +2,31 @@
...
+
+- CPU
+  - 12 cores / 24 threads, or more
+  - 2.8GHz, or faster
+  - AVX2 instruction support
```
Comment (sakridge) [inline docs/src/running-validator/validator-reqs.md:10]: If we want to be very generic, thread count is not really important SMT is generally 10-15% at best improvement. Clock speed may not be that important if IPC is high. AVX2 not required, just need to build from source.

Practically, these are good suggestions for most shipping x86 chips though.

File: docs/src/running-validator/validator-reqs.md (hunk 2 above)
Comment (t-nelson) [reply docs/src/running-validator/validator-reqs.md:41]: And yet, I missed the date in the previous line :upside_down_face:

File: docs/src/running-validator/validator-reqs.md (hunk 3 above)
Comment (t-nelson) [reply docs/src/running-validator/validator-reqs.md:18]: I'll add a parenthetical spelling it out to the first use

File: docs/src/running-validator/validator-reqs.md (hunk 4 above)
Comment (t-nelson) [reply docs/src/running-validator/validator-reqs.md:5]: The Samsung 860 EVO line includes an SATA SKU.  This change set was actually inspired by a new validator being bitten by this prescription :face_with_head_bandage:

**Review by sakridge — APPROVED** [review]

File: docs/src/running-validator/validator-reqs.md (hunk 5 above)
Comment (t-nelson) [reply docs/src/running-validator/validator-reqs.md:10]: > If we want to be very generic, thread count is not really important SMT is generally 10-15% at best improvement.

I mainly mentioned threads here for the clarity that by cores, we mean physical cores.

> Clock speed may not be that important if IPC is high.

2.8GHz was chosen being what someone asking on Discord is likely to get

> AVX2 not required, just need to build from source.

I'll add a note that this is purely a requirement of using our release binaries
 
> Practically, these are good suggestions for most shipping x86 chips though.

:+1:  That's what I was going for!  The guiding principle was, "what have people been complaining about or championing on Discord for the last 2 mo" :joy:

File: docs/src/running-validator/validator-reqs.md [hunk 6]
```diff
@@ -2,35 +2,32 @@
...
-| RAM                 | 32GB                                                  | 64GB                   | 128GB                  |                                                                                        |
-| Ledger Drive        | Samsung 860 Evo 2TB                                   | Samsung 860 Evo 4TB    | Samsung 860 Evo 4TB    | Or equivalent SSD                                                                      |
-| Accounts Drive\(s\) | None                                                  | Samsung 970 Pro 1TB    | 2x Samsung 970 Pro 1TB |                                                                                        |
-| GPU                 | Nvidia 1660ti                                         | Nvidia 2080 Ti         | 2x Nvidia 2080 Ti      | Any number of cuda-capable GPUs are supported on Linux platforms.                      |
+  - Not strictly necessary at this time
+  - Motherboard and power supply speced to add one or more high-end gaming GPUs
```
Comment (danpaul000) [inline docs/src/running-validator/validator-reqs.md:29]:
```suggestion
  - Motherboard and power supply speced to add one or more high-end GPUs
```

Comment (t-nelson) [reply docs/src/running-validator/validator-reqs.md:29]: Sorry @danpaul000, the :robot: beat me to these.  I'll add in a followup

File: docs/src/running-validator/validator-reqs.md [hunk 7]
```diff
@@ -2,35 +2,32 @@
...
+in the future suggested
```
Comment (ryoqun) [inline docs/src/running-validator/validator-reqs.md:30]: @t-nelson So lame me... I was completely forgetting this was the perfect time to add mention of required network bandwidth for both uplink and downlink...  Remember the support ticket.. ;)

**Pull request merged**